packages/core/test/popover/popoverTests.tsx

```diff
@@ -24,9 +24,10 @@ import { Portal } from "../../src/index";
 describe("<Popover>", () => {
     let testsContainerElement: HTMLElement;
     let wrapper: IPopoverWrapper;
-    const onInteractionSpy = sinon.spy();
+    let onInteractionSpy: sinon.SinonSpy;
```

Comment:
what's up with the unrelated changes in this file? please revert.

Reply:
ah semantic merge conflicts. i made this change in two PRs, slightly differently it seems.